docs/man.md

```diff
@@ -18,12 +18,10 @@ The `qpc-tools` package is used to configure the Quipucords server and command l
 The following sections describe these commands, their subcommands, and their options in more detail. They also describe additional tasks that are not highlighted in the previous list of major workflow tasks.
 
 ## Server
-This section describes various `qpc-tools` commands for installing and configuring the Quipucrods server.
```

Comment:
Bahahah good catch there ash. Quipucrods would for sure not been picked up in the downstream sed replace.

Reply:
😄